doc/tutorial.rst

```diff
@@ -294,38 +271,69 @@ Lets' start by generating and showing the data.
 
 .. code:: python
 
-    X_train, y_train, X_test, y_test, y_mesh = generate_onedimensional_data(
-        x_sinx, distrib='normal', n_samples=300, mu=0, sig=2, sigfactor=4, sig_noise=0
+    def get_1d_data_with_normal_distrib(funct, mu, sig, n_samples, sig_noise):
+        np.random.seed(59)
+        X_train = npr.normal(mu, sig, n_samples)
+        X_test = np.arange(mu-4*sig, mu+4*sig, sig/20.)
```

Comment:
privilégier `sigma` plutôt `sig`.

"Explicit is better than implicit"
The Zen of Python